openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1389,3 +1397,109 @@ def verify_setup(self, prefix, timeout=300, wait_step=10):
                 h_vol_list, g_vol_list, pv_vols))
 
         self.verify_pods_are_running()
+
+    def create_heketi_volumes_in_batch(
```

Comment:
There is already an open patch for this method https://github.com/gluster/glusterfs-containers-tests/pull/33

Reply:
This was due to incorrect rebase